lib/RuntimeNode43.js

```diff
@@ -31,6 +31,12 @@ module.exports = function(S) {
               functionFile    = func.getRootPath(handlerArr[0] + (func.handlerExt || '.js')),
               functionHandler = handlerArr[1];
 
+            // Switch to the root of the lambda package from the handler property
+            const handlerFullPath = func.getRootPath(_.last(func.handler.split('/'))).replace(/\\/g, '/');
```

Comment:
You should use 'path' or a similar path handling library here, that defines the PATH_SEPARATOR constant. We should not expect the separator to be '/'. On Windows the replace will fail.